### ISSUE TITLE
Filtering column in source file connector shuffles column ordering.

### ISSUE THREAD

**Issue by rishabh-cldcvr** [airbytehq/airbyte#7045]
## Enviroment
- **Airbyte version**:  0.30.16-alpha
- **OS Version / Instance**: macOS
- **Deployment**: Docker or Kubernetes deploy env
- **Source Connector and version**: airbyte/source-file:0.2.6
- **Severity**: Medium
- **Step where error happened**: Output of connector

## Current Behavior
When using source-connector image for file, it shuffles columns position in the output.

## Expected Behavior
keep original ordering of columns.

<details>
<summary>LOG</summary> 

```
Original record : 
{"index": 248, "join_date": "2020-03-15", "first_name": "Jia Yu ", "last_name": "Zeng", "address": "Blk 368A Jalan Zemlak Hill SINGAPORE 257066", "gender": "FEMALE", "jae_code": "C31"}

Output connector record from image:
{"type": "RECORD", "record": {"stream": "s3_test_ingest_2021_Oct_13", "data": {"last_name": "Zeng", "jae_code": "C65", "first_name": "Jia Yu ", "index": 0, "join_date": "2020-02-15", "gender": "FEMALE", "address": "Blk 368A Jalan Zemlak Hill SINGAPORE 257066"}, "emitted_at": 1634204204000}}

```

</details>

## Steps to Reproduce
1. run latest airbyte/source-file:0.2.6
2. check container output

## Are you willing to submit a PR?
I have fix ready and tested it out in local and it works great.

**Comment by marcosmarxm** [comment]
Hello @rishabh-cldcvr sorry for not replying here. Did you finish the change? Let me know if you need any assistance to submit the PR.

**Comment by midavadim** [comment]
I was not able to reproduce this problem with the latest version of source-file.

I tried to sync json/jsonl from local file and from http source.
I also tried to sync exactly the same data sample mentioned in issue description:
{"index": 248, "join_date": "2020-03-15", "first_name": "Jia Yu ", "last_name": "Zeng", "address": "Blk 368A Jalan Zemlak Hill SINGAPORE 257066", "gender": "FEMALE", "jae_code": "C31"}

Columns order is kept in the same order each time.
